.devcontainer/docker-compose.yml

```diff
@@ -0,0 +1,31 @@
+version: "3"
```

Comment:
> Do we need a new Docker compose file?

Yes, I think so because [there's no good way to set the environment variables](https://stackoverflow.com/questions/70227082/configuring-environment-variables-prior-to-creation-of-dev-container) that cpp-debug.yml reads to build the container.